src/features/cartographie/components/lieux-mediation-numerique-details/impression-choice-modal/impression-choice-modal.component.html

```diff
@@ -23,7 +23,7 @@ <h5 class="h5 fw-bold text-primary">Vous êtes médiateur numérique ?</h5>
   <button
     type="button"
     (click)="print.emit()"
-    class="btn bg-white mb-3 px-lg-3 px-2 py-3 rounded-0 d-flex position-relative w-100">
+    class="btn bg-white mb-4 px-lg-3 px-2 py-3 rounded-0 d-flex position-relative w-100 mt-4">
```

Comment:
```suggestion
    class="btn bg-white px-lg-3 px-2 py-3 rounded-0 d-flex position-relative w-100 my-4">
```
`mt-4` + `mb-4` = `my-4`. Comme `my-*` c'est à la fois en haut et en bas, tu peux fusionner les deux classes en une seule.